src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests/Builders/HeathCheckStrategyBuilder.cs

```diff
@@ -0,0 +1,41 @@
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.HealthChecks.Strategies;
+using Moq;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests.Builders;
+
+public class HeathCheckStrategyBuilder
```

Comment:
Looks like 'L' key might be broken 😆.  Throughout this class 'health' is 'heath'.